jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/SysinfoCpuTime.java

```diff
@@ -29,19 +29,23 @@
 	private int numberOfCpus; /* should be at least 1 */
 	private int status;
 	private long processCpuTime; /* in ns. The process Cpu Time, used for getProcessCpuLoad() */
-
+	private int cpuLoad;
```

Comment:
Can we get some Javadoc documentation here? It is certanly not obvious why we have both `cpuLoad` and `processCpuTime` and when one should use one vs. the other. This would be a great place to mention the meaning of the `-1` `cpuLoad` value.

Reply:
Change applied in 5edae7db32e8a09931cf979411bb18bf46ad95b2